era5_land_forcing_data2021/README.md

```diff
@@ -0,0 +1,24 @@
+# Clima Land forcing data  - ERA5 reanalysis data for 2021
+This file holds one year of hourly data of a subset of ERA5 output fields, at
+approximately 1 degree of resolution. The fields are
+- Latitude ("lat")
+- Longitude ("lon")
+- Time ("time")
+- Temperature at 2 meters (lon, lat, time; "t2m")
+- Downwelling SW flux accumulation at surface (lon, lat, time, "ssrd")
+- Downwelling LW flux accumulation at surface (lon, lat, time, "strd")
+- Neutral wind at 10m, u-component (lon, lat, time, "u10n")
+- Neutral wind at 10m, v-component (lon, lat, time, "v10n")
+- Surface air pressure (lon, lat, time, "sp")
+- Dew point temperature at 2 meters (lon, lat, time, "d2m")
+- Total accumulated precipitation (lon, lat, time, "tp")
+- Accumulated snowfall (lon, lat, time, "sf")
+- Leaf area index, high vegetation (lon, lat, time, "lai_hv")
+- Leaf area index, low vegetation (lon, lat, time, "lai_lv")
+
+# Attribution
```

Comment:
@braghiere do you know if this is enough regarding citation of ERA5? I followed their instructions here:
https://confluence.ecmwf.int/display/CKB/Use+Case+2%3A+ERA5+hourly+data+on+single+levels+from+1940+to+present

Reply:
A more formal academic citation used for ERA5 is:

Hersbach, H., Bell, B., Berrisford, P., Hirahara, S., Horányi, A., Muñoz-Sabater, J., Nicolas, J., Peubey, C., Radu, R., Schepers, D., Simmons, A., Soci, C., Abdalla, S., Abellan, X., Balsamo, G., Bechtold, P., Biavati, G., Bidlot, J., Bonavita, M., … Thépaut, J.-N. (2020). The ERA5 global reanalysis. Quarterly Journal of the Royal Meteorological Society, 146(730), 1999–2049. https://doi.org/10.1002/QJ.3803